Doppler.ReportingApi/Controllers/DetailController.cs

```diff
@@ -10,6 +10,20 @@ namespace Doppler.ReportingApi.Controllers
     [ApiController]
     public class DetailController
     {
+        [HttpGet]
+        [Route("{accountName}/details/tipsandnotifications")]
```

Comment:
En general tratamos de utilizar lower-case y slugs en nuestras nuevas URLs, que opinás de esta?

```suggestion
        [Route("{accountName}/details/tips-and-notifications")]
```

Por otro lado, para tener algo de contexto, te parece hacer un pequeño documento con un draft de como crecería esta API, con borradores de las URLs y de los schemas de los recursos? En base a eso podemos tener una breve charla entre los interesados para entender mejor como va a crecer esta API.